bitbots_docs/docs/manual/hardware/electronics.rst

```diff
@@ -0,0 +1,47 @@
+==========
+Elektronik
```

Comment:
```suggestion
Electronics
```
When applying this, the "=" above and below also need to be expanded by one "=" each.